jkube-kit/common/src/test/java/org/eclipse/jkube/kit/common/util/EnvUtilTest.java

```diff
@@ -23,15 +23,320 @@
 import java.nio.charset.StandardCharsets;
 import java.nio.file.Files;
 import java.nio.file.Path;
+import java.util.ArrayList;
+import java.util.Collections;
 import java.util.Date;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.Properties;
 
+import static org.eclipse.jkube.kit.common.util.EnvUtil.firstRegistryOf;
+import static org.eclipse.jkube.kit.common.util.EnvUtil.isWindows;
 import static org.eclipse.jkube.kit.common.util.EnvUtil.loadTimestamp;
 import static org.eclipse.jkube.kit.common.util.EnvUtil.storeTimestamp;
+import static org.hamcrest.MatcherAssert.assertThat;
 import static org.hamcrest.core.IsEqual.equalTo;
-import static org.junit.Assert.assertThat;
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotEquals;
+import static org.junit.Assert.assertNull;
+import static org.junit.Assert.assertTrue;
+
 
 public class EnvUtilTest {
 
+    @Test
+    public void testConvertTcpToHttpsUrl() {
+        // Given
+        String urlWithHttpsPort = "tcp://0.0.0.0:2376";
+        // When
+        String result1 = EnvUtil.convertTcpToHttpUrl(urlWithHttpsPort);
+        // Then
+        assertEquals("https://0.0.0.0:2376", result1);
+    }
+
+    @Test
+    public void testConvertTcpToHttpUrl() {
+        // Given
+        String urlWithHttpPort="tcp://0.0.0.0:2375";
+        // When
+        String result2 = EnvUtil.convertTcpToHttpUrl(urlWithHttpPort);
+        // Then
+        assertEquals("http://0.0.0.0:2375", result2);
+    }
+
+    @Test
+    public void testExtractLargerVersionWhenBothNull(){
+        //Given
+        // Both version are null
+        //When
+        //Then
+        assertNull(EnvUtil.extractLargerVersion(null,null));
```

Comment:
Please, remove the comments:
```suggestion
        assertNull(EnvUtil.extractLargerVersion(null,null));
```
Or replace by  declarative code:
```suggestion
        //Given
        var v1 = null;
        var v2 = null;
        //When
        var result = EnvUtil.extractLargerVersion(v1, v2);
        //Then
        assertNull(result);
```